src/redux/walletconnect.ts

```diff
@@ -198,15 +198,13 @@ const WALLETCONNECT_ADD_URI = 'walletconnect/WALLETCONNECT_ADD_URI';
  */
 const getNativeOptions = async () => {
   const language = 'en'; // TODO use lang from settings
-  let token = null;
-  try {
-    token = await getFCMToken();
-  } catch (error) {
+  const token = await getFCMToken();
+
+  if (!token && !IS_DEV) {
```

Comment:
Only logs in prod since it will fail every time locally (same handling in other file)